docs/docs/getting-started.server.mdx

```diff
@@ -506,7 +506,47 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 through webpack loader syntax in imports.
 
 Install the webpack loader [`@mdx-js/loader`][mdx-loader].
-There is no need to configure it.
+
+To enable direct MDX imports w/o `!@mdx-js/loader!` prefix, the loader can
+further be added to webpack’s config, using a
+[`react-scripts`](http://github.com/facebook/create-react-app/tree/main/packages/react-scripts)
+rewiring tool, e.g. [CRACO](http://github.com/gsoft-inc/craco).
```

Comment:
```suggestion
rewiring tool, such as [CRACO](http://github.com/gsoft-inc/craco).
```

`such as`, compared to a latin abbreviation, is often easier to understand.

Are the popular and viable others?

Reply:
> Are the popular and viable others?

Popular, yes: https://github.com/timarney/react-app-rewired – the standard for years! – only not compatible yet w/CRA 5; which might not change, given its poor recent maintenance … So actually: CRACO is the only viable one. I'll further rephrase that section …